netci.groovy

```diff
@@ -757,14 +778,18 @@ def static isValidPrTriggeredInnerLoopJob(os, architecture, configuration, isBui
     return true
 }
 
+// This means the job builds and runs the 'Pri0' test set. This does not mean the job is 
+// scheduled with a default PR trigger despite the correlation being true at the moment.
+def static isPri0TestScenario(def scenario) {
```

Comment:
We've been moving away from the "test priority" terminology to the simpler and adequate "innerloop" versus "outerloop."  I realize the command line switches still use the older term, but I'd rather not re-introduce usages we've already consciously removed.